pages/data-migration/migrate-from-rdbms-directly.md

```diff
@@ -0,0 +1,281 @@
+However, in the first argument we will provide a query that selects columns from the table.
+
+```cypher
+CALL migrate.postgresql('SELECT * FROM BandMemberships' , {
+  user:'postgres',
+  password:'xxxxxx',
+  host:'localhost',
+  database:'postgresdb'
+})
+YIELD row
+RETURN row.band_id AS band_id, row.musician_id AS musician_id;
+```
+
+![](/pages/data-migration/migrate-from-rdbms-directly/migrate-from-rdbms-directly-relationships-row.png)
+
+This query is useful as sometimes you don't need all the columns to be migrated to Memgraph, you only need a fraction
+of them. Although you can omit creation of properties when the row is received, this approach consumes less memory on
+the row level. We can create the relationships by matching the source and destination node we migrated previously. Now,
+the indices will play a big role, as the graph can be connected effectively and with optimal performance.
+
+```cypher
+CALL migrate.postgresql('SELECT * FROM BandMemberships' , {
+  user:'postgres',
+  password:'xxxxxx',
+  host:'localhost',
+  database:'postgresdb'
+})
+YIELD row
+MATCH (m:Musician {id: row.musician_id}), (b:Band {id: row.band_id})
+CREATE (m)-[:PERFORMS_IN]->(b);
+```
+
+After migrating the relationships, we can see how our graph looks like:
+
+```cypher
+MATCH (n)-[r]->(m) RETURN n, r, m;
+```
+
+![](/pages/data-migration/migrate-from-rdbms-directly/migrate-from-rdbms-directly/connected-graph.png)
+
+### 5. Switch back to in-memory transactional mode
+If we want to give back the transactional guarantees, we can do so by switching back from
+analytical to transactional mode with the query:
+```cypher
+STORAGE MODE IN_MEMORY_TRANSACTIONAL;
+```
+
+
+## Benefits of direct migration from RDBMS to Memgraph
+We have shown how Memgraph can, with the easy integration of MAGE modules, migrate from an external
+source to its graph storage in a matter of few queries. That offers a significant improvement in the
+ease of use than the CSV import we have provided previously. Users are encouraged to see for themselves
+which method suits them better.
+
+## Frequently asked questions
+
+### What are some of the best practices to lower memory when migrating data?
+We have already mentioned the in-memory analytical mode that has less memory overhead since it 
+strips Memgraph of some ACID guarantees.
+
+We will look at some other techniques which we use to lower memory:
+1. Omitting unnecessary columns when migrating
+2. Appropriate data type conversion
+
+####  1. Ommitting unnecessary columns when migrating
+Most users will want to migrate full rows to Memgraph, which can sometimes be inefficient.
+The columns in Postgres are meant to be stored on
+disk, and therefore users can add new columns which can be big in size. This mainly is a concern
+for unnecessary date strings and urls. 
+
+This can be optimized by processing the minimal set of necessary columns.
```

Comment:
Say more to help the user decide which columns are necessary. Perhaps something about thinking about which graph traversal algorithms are going to be used, and the inputs they need.